app/react/src/server/config.js

```diff
@@ -73,7 +73,10 @@ export default function(configType, baseConfig, configDir) {
       ...config.module,
       // We need to use our and custom rules.
       ...customConfig.module,
-      rules: [...config.module.rules, ...(customConfig.module.rules || [])],
+      rules: [
+        ...config.module.rules,
+        ...((customConfig.module && customConfig.module.rules) || []),
```

Comment:
@shilman I don't believe so. `customConfig.module` should be either an object or undefined. In the case of an object, the application of the object spread operator is fine. In the case of undefined, it's also fine. See [Null/Undefined Are Ignored](https://github.com/tc39/proposal-object-rest-spread/blob/master/Spread.md).

Testing with `npm link`, I have not observed additional issues.